templates.dist/fields.php

```diff
@@ -223,3 +223,94 @@ function renderRegionalAddition(string $inputName, array $selectedValue, bool $m
 		return $output;
 	}
 }
+
+
+if (!function_exists('renderParkingLot')) {
+	function renderParkingLot(array $parkingArray, string $language, string $locale = 'de_DE'): array
+	{
+		$messages = [];
+		foreach ($parkingArray as $key => $parking) {
+			if (!$parking['Count']) {
+				continue;
+			}
+			/* translators: 1: Name of parking lot, 2: Price */
+			$element = sprintf(__('%1$s at %2$s', 'onoffice'), getParkingName($key, $parking['Count']), formatPriceParking($parking['Price'], $language, $locale));
```

Comment:
I have updated the requirements:
Please display to different texts depending on whether it is a single item or multiple (the "each" only appears when there is more than one):
- 1 garage at €500 (purchase)
- 10 free parking lots at €45.99 each (rent)

Reply:
sorry, i will fix it as soon !